cloudsync/sync/state.py

```diff
@@ -107,16 +110,28 @@ def __setattr__(self, k, v):
             object.__setattr__(self, k, v)
             return
 
+        if k == "exists":
+            if v == CORRUPT and not self.is_corrupt:
+                self._saved_exists = self.exists
+                self._exists = v
```

Comment:
we don't want to call parent update.... why?

Reply:
because nothing actually got updated